engine/model.go

```diff
@@ -85,9 +85,40 @@ func (a *Address) Equals(o Address) bool {
 	return a.Network == o.Network && a.Address == o.Address
 }
 
+// Sets up OCSP stapling
+type OCSPSettings struct {
+	Enabled            bool
+	Period             string
+	Responders         []string // optional responder
```

Comment:
It's not obvious to me what "responder" means here. Can you add a short comment to clarify what this is?